src/gwproto/types/spaceheat_node_gt.py

```diff
@@ -1,16 +1,16 @@
-"""Type spaceheat.node.gt, version 100"""
+"""Type spaceheat.node.gt, version 110"""
 
 from typing import Literal, Optional
 
 from pydantic import BaseModel, Field
 
 from gwproto.enums import ActorClass, Role
-from gwproto.property_format import LeftRightDotStr, UUID4Str
+from gwproto.property_format import SpaceheatName, UUID4Str
 
 
 class SpaceheatNodeGt(BaseModel):
     ShNodeId: UUID4Str
-    Alias: LeftRightDotStr
+    Alias: SpaceheatName
```

Comment:
IIRC this will mean we have to update scada for this change in order to verify the other changes in this PR. For example without this change and the removal of cacs in jm/reduce there's a change that scada would "just work" with these changes? In any case we have to update scada at some point, so it might not matter when.

Reply:
What I discovered is that we need CACS_BY_MAKE_MODEL in place before removing the various cacs, to replace the `cacs_by_type` with `cac_by_make_model`  in the scada code